src/js/components/homepage/LinksSectionBox.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * LinksSectionBox.jsx
+ * Created by Marco Mendoza 03/16/2017
+ **/
+
+import React from 'react';
+
+import * as HomeIcons from 'components/sharedComponents/icons/home/HomeIcons';
+import * as Icons from 'components/sharedComponents/icons/Icons';
+
+const propTypes = {
+    icon: React.PropTypes.string,
+    text: React.PropTypes.string,
+    linkText: React.PropTypes.string,
+    linkUrl: React.PropTypes.string
+};
+
+export default class LinksSectionBox extends React.Component {
+
+    render() {
+        let icon = this.props.icon;
+
+        switch (icon) {
```

Comment:
For the purposes of keeping `render()` small, let's move this `switch` statement into it's own function, above `render()` (something like `generateIcon()`).